servers/src/test/java/tachyon/master/file/FileSystemMasterTest.java

```diff
@@ -62,6 +64,7 @@
   private static final TachyonURI ROOT_FILE_URI = new TachyonURI("/file");
   private static final TachyonURI TEST_URI = new TachyonURI("/test");
   private static CreateOptions sNestedFileOptions;
+  private static long sOldTtlIntervalMs;
```

Comment:
Our style guide says we should Ttl. I'll create a ticket + followup followup PR for it.